src/components/BottomNav/BottomNav.js

```diff
@@ -25,10 +25,10 @@ export function BottomNav() {
       </Link>
       <Link to="/dashboard">
         <Ripple classes="bottomNavButton">
-          <Thunder active={location.pathname === '/dashboard' ? true : false} />
+          <Thunder active={location.pathname === '/matchPage' ? true : false} />
           <p
             className={
```

Comment:
I was actually thinking that the center "thunder" icon would take you back to the dashboard and then you press "start" to start swiping.  I'm looking into pre-loading images so there is not lag and I was hoping to use the dashboard page for that.

Reply:
This I havent touched